src/main/java/org/folio/inventory/dataimport/consumers/DataImportKafkaHandler.java

```diff
@@ -61,6 +63,8 @@
 
   private static final Logger LOGGER = LogManager.getLogger(DataImportKafkaHandler.class);
   private static final String RECORD_ID_HEADER = "recordId";
+  private static final String CHUNK_ID_HEADER = "chunkId";
+  private static final String JOB_EXECUTION_ID_HEADER = "jobExecutionId";
```

Comment:
Here is job execution id header from lower case, and in MarcBibInstanceHridSetKafkaHandler its from upper case. In other classes the same, can we unify it in the same way(use the same case or extract as constant in some utility class)?

Reply:
@SerhiiNosko Yes, thank you! I missed to retrieve jobExecutionId from headers, not from eventPayload in DataImportKafkaHandler. I removed this constant from that handler.